pkg/cmd/pipelinerun/log_reader.go

```diff
@@ -90,12 +92,18 @@ func (lr *LogReader) readLiveLogs(pr *v1alpha1.PipelineRun) (<-chan Log, <-chan
 		}
 
 		wg.Wait()
+
+		if pr.Status.Conditions[0].Status == corev1.ConditionFalse {
```

Comment:
Need to query `PippelineRun` again because `PipelineRun` state is not equal to the actual `PipelineRun` state

Reply:
Thanks, New implementation uses watch api.